_run-with-mounted-folder.ps1

```diff
@@ -0,0 +1,4 @@
+Write-Host "When docker has run an you are at bash inside container, you can do for example:"
+Write-Host "python default.py -u https://webperf.se -t 26 -o reports/webperf-se-t-26.json -r"
```

Comment:
@krompaco please use `-i sites.json` instead of `-u  https://webperf.se`.

Reply:
These are just a sort on-the-fly documentation... Added sites.json as an example too.